content/Watsonx/WatsonxData/106.md

```diff
@@ -0,0 +1,202 @@
+---
+title: '106: Federated Queries'
+timeToComplete: 45
+updated: 2023-09-19
+---
+
+## Federated Queries
+
+Unlike traditional database systems, Presto doesn’t have its own native database storage. Instead, Presto supports separation of compute and storage, with dozens of connectors that let Presto access data where it lives – which could be in relational databases, NoSQL databases, data warehouses, data lakes, data lakehouses, and more.
+
+Although Presto supports a wide range of connectors, watsonx.data officially only supports a subset of them. This is because IBM wants to ensure quality, performance, and security of connectors before adding support (which may require updating connector source code to do this). More connectors will be added over time.
+
+The most current list of connectors and the types SQL statements supported can be found [here](https://www.ibm.com/docs/en/watsonxdata/1.0.x?topic=watsonxdata-supported-sql-statements). The list of supported connectors will grow over time.
+
+In this section you will combine data from watsonx.data’s object storage with data in Db2 and PostgreSQL databases. To avoid you having to provision these databases yourself, they’ve been installed in the same VM as watsonx.data and are pre-populated with data.
+
+1. Select the **Infrastructure manager** icon from the left-side menu.
+
+2. Click the **Add component** dropdown menu at the right-side of the screen. Select **Add Database**.
+
+  ![](./images/106/add-database.png)
+
+3. In the **Add database** pop-up window, select/enter the following pieces of information in the **1. Database definition** section:
+
+
+            • Database type:    IBM Db2
```

Comment:
Bullet points should be separated by newline characters so that they appear on separate lines.

Reply:
Added bullet points